module/component/component.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/onflow/flow-go/module/util"
 )
 
+// ErrComponentShutdown is returned by a component which has already been shut down.
+var ErrComponentShutdown = fmt.Errorf("component has already shut down")
```

Comment:
again, don't believe we handle it somewhere and don't see a practical usage for it, I mean even if we handle it we would most likely just ignore it either way?